MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -161,12 +164,12 @@ private async Task FetchDataFromTags(Audio audio, Model.MediaInfo.MediaInfo medi
             var libraryOptions = _libraryManager.GetLibraryOptions(audio);
             Track track = new Track(audio.Path);
 
-            // ATL will fall back to filename as title when it does not understand the metadata
-            if (track.MetadataFormats.All(mf => mf.Equals(ATL.Factory.UNKNOWN_FORMAT)))
-            {
-                track.Title = mediaInfo.Name;
-            }
+            // Current version of ATL split some ID3v2.2/2.3 tags by '/' which is not desired.
+            var shouldWorkaroundAtlId3V22Split = track.MetadataFormats
```

Comment:
We can also check `libraryOptions.UseCustomTagDelimiters` if we want to preserve the old behavior without custom tag delimiters.